WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/File List/ProductDownloadListViewModel.swift

```diff
@@ -50,6 +54,10 @@ final class ProductDownloadListViewModel: ProductDownloadListViewModelOutput {
     private(set) var downloadLimit: Int64
     private(set) var downloadExpiry: Int64
 
+    var bottomSheetActions: [DownloadableFileFormBottomSheetAction] {
+        return [.fromDevice, .fromWordPressMediaLibrary, .fromFileURL]
```

Comment:
nit: just to look more concise, but up to you!
```suggestion
        [.fromDevice, .fromWordPressMediaLibrary, .fromFileURL]
```